### ISSUE TITLE
stats/percentage selector layout

### ISSUE THREAD

**Issue by patudom** [cosmicds/hubbleds#422]
Noting cosmetic issues here for someone to fix (roughly in order of importance) once all the functionality is in place. Top image is the voila layout. Bottom is the solara.

- [x] Add definitions & example graphs for stats component  (after this is complete, label can be downgraded to "lower priority")
- [x]  Histogram viewer needs a blue title bar
- [x] Place selector components side by side so they take up less vertical space
- [x] Selector components should get a dark blue background and have a gray outline.
- [x] Improve styling of dialogs that pop up for stats/percentage descriptions.

<img width="1057" alt="Screenshot 2024-06-17 at 11 03 21 AM" src="https://github.com/cosmicds/hubbleds/assets/12750048/0578197e-05d0-4798-a08b-34814f3fd9b3">
<img width="1090" alt="Screenshot 2024-06-17 at 10 41 42 AM" src="https://github.com/cosmicds/hubbleds/assets/12750048/654b9dbd-1179-4ca6-af48-9d8da42547d4">
